octoprint_mrbeam/util/version_comparator.py

```diff
@@ -34,3 +34,12 @@ def get_comparator(comparision_string, comparision_options):
         for item in comparision_options:
             if item.identifier == comparision_string:
                 return item
+
+
+COMPARISON_OPTIONS = [
```

Comment:
I would place this at the beginning of the file

Reply:
This shows an error when placed at the top. The IDE marks the class 'VersionComparator' as not defined yet

Reply:
okay